### PR TITLE
changed docker command

### PR DESCRIPTION
**Pull request by mbernasocchi**
changed docker pull osgeo/proj.4 to changed docker pull osgeo/proj

### CODE REVIEW

**Comment by mbernasocchi** [comment]
@hobu ups, got carried away :)

File: docs/source/install.rst [hunk 1]
```diff
@@ -26,13 +26,12 @@ from the ``conda-forge`` channel::
 
     conda install -c conda-forge proj4
 
-Docker
```
Comment (hobu) [inline docs/source/install.rst]: Why was this line removed?

**Review by hobu — APPROVED** [review]

**Comment by kbevers** [comment]
Cherry-picked into 5.2 - changes will be published soon.

Thanks for fixing this.
